snooty/semanticparser.py

```diff
@@ -9,7 +9,9 @@ def __init__(self) -> None:
     def run(self, pages: Dict[FileId, Page]) -> Dict[str, SerializableType]:
         # Specify which transformations should be included in semantic postprocessing
         functions: List[Callable[[Dict[FileId, Page]], Dict[str, SerializableType]]] = [
-            self.slug_title
```

Comment:
// TODO: wrap this in a function so that we might toggle what semantic parse layers are exposed based on an environment setting. [Useful for when we support additional transforms for REST retrieved content]